<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Project Sweat</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="grafik">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>130</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QComboBox" name="change_prod">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>630</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="to_grafik">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>320</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>График</string>
    </property>
   </widget>
   <widget class="QPushButton" name="to_table">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>360</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Ваши товары</string>
    </property>
   </widget>
   <widget class="QPushButton" name="to_add">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>400</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить товар</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reboot">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>690</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Обновить</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
